Dockerfile.perses

```
@@ -0,0 +1,33 @@
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.23 AS build-env
```

We are missing the Perses UI, in the dev preview is important as we don't have dashboard customization yet:

```suggestion
FROM registry.redhat.io/ubi9/nodejs-18:latest AS web-builder

WORKDIR /web

COPY Makefile Makefile
COPY ui ui

USER 0

RUN make build-ui

FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_8_golang_1.23 AS build-env
```